--- ./Makefile.org	Sun Aug 11 22:45:52 2002
+++ ./Makefile	Tue Sep 10 15:13:50 2002
@@ -8,12 +8,14 @@
 
 ####### Compiler, tools and options
 
+RINGDIR  = /usr/local/lib/xdrawchem
+XDCSERVER= herger.ibb.gatech.edu
 CC       = gcc
 CXX      = g++
 LEX      = flex
 YACC     = yacc
-CFLAGS   = -pipe -Wall -W -g  -DUNIX -DRINGHOME=\"/usr/local/lib/xdrawchem\" -DXDC_SERVER=\"herger.ibb.gatech.edu\"
-CXXFLAGS = -pipe -Wall -W -g  -DUNIX -DRINGHOME=\"/usr/local/lib/xdrawchem\" -DXDC_SERVER=\"herger.ibb.gatech.edu\"
+CFLAGS   = $(OPT) -pipe -Wall -W -g  -DUNIX -DRINGHOME=\"$(RINGDIR)\" -DXDC_SERVER=\"$(XDCSERVER)\"
+CXXFLAGS = $(OPT) -pipe -Wall -W -g  -DUNIX -DRINGHOME=\"$(RINGDIR)\" -DXDC_SERVER=\"$(XDCSERVER)\"
 LEXFLAGS = 
 YACCFLAGS= -d
 INCPATH  = -I$(QTDIR)/include -I$(QTDIR)/mkspecs/default
@@ -33,6 +35,7 @@
 DEL_FILE = rm -f
 DEL_DIR  = rmdir
 MOVE     = mv
+QMAKECONF= $(QTDIR)/mkspecs/default/qmake.conf 
 
 ####### Output directory
 
@@ -233,9 +236,9 @@
 		moc_peptidebuilder.o \
 		moc_smilesdialog.o
 DIST	   = 
-QMAKE_TARGET = xdrawchem
-DESTDIR  = /usr/local/bin/
-TARGET   = /usr/local/bin/xdrawchem
+QMAKE_TARGET	= xdrawchem
+BINDIR	= /usr/local/bin/
+TARGET	= xdrawchem
 
 first: all
 ####### Implicit rules
@@ -262,15 +265,14 @@
 all: Makefile $(TARGET)
 
 $(TARGET): $(UICDECLS) $(OBJECTS) $(OBJMOC) 
-	test -d /usr/local/bin/ || mkdir -p /usr/local/bin/
 	$(LINK) $(LFLAGS) -o $(TARGET) $(OBJECTS) $(OBJMOC) $(LIBS)
 
 mocables: $(SRCMOC)
 
-$(MOC): 
-	( cd $(QTDIR)/src/moc ; $(MAKE) )
+$(MOC):
+	( exit 0 )
 
-Makefile: xdrawchem.pro  $(QTDIR)/mkspecs/default/qmake.conf 
+Makefile: xdrawchem.pro $(QMAKECONF)
 	$(QMAKE) xdrawchem.pro
 qmake: 
 	@$(QMAKE) xdrawchem.pro
@@ -1103,83 +1105,85 @@
 ####### Install
 
 install_ringfile: 
-	@test -d $(INSTALL_ROOT)/usr/local/lib/xdrawchem/ || mkdir -p $(INSTALL_ROOT)/usr/local/lib/xdrawchem/
-	-$(COPY_FILE) "ring/6ring_boat.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/6ring_chair.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/adenine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/alanine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/anthracene.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/arginine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/asparagine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/aspartic_acid.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/benzene.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/benzene.png" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/biphenyl.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/boc.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/boc.cml~" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/cyclobutane.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/cycloheptane.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/cyclohexane.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/cyclohexane.png" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/cyclooctane.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/cyclopentadiene-sp.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/cyclopentadiene.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/cyclopentane.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/cyclopentane.png" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/cyclopropane.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/cysteine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/cytosine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/d-fructose.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/d-glucose.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/dabcyl.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/dabsyl.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/dansyl.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/deoxyribose.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/edans.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/fmoc.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/glutamic_acid.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/glutamine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/glycine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/guanine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/histidine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/imidazole.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/imidazole.png" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/indole.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/isoleucine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/leucine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/lysine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/methionine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/naphthalene.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/nitrophenylalanine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/phenylalanine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/proline.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/purine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/pyrimidine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/ribose.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/serine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/statine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/steroid.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/threonine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/thymine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/tryptophan.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/tyrosine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/uracil.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/valine.cml" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/xdrawchem_de.qm" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/xdrawchem_en.qm" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/xdrawchem_es.qm" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/xdrawchem_fr.qm" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/xdrawchem_it.qm" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/xdrawchem_nl.qm" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/xdrawchem_pl.qm" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/xdrawchemrc" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
-	-$(COPY_FILE) "ring/xdrawchemrc~" "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
+	@test -d $(DESTDIR)$(RINGDIR) || mkdir -p $(DESTDIR)$(RINGDIR)
+	-$(COPY_FILE) "ring/6ring_boat.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/6ring_chair.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/adenine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/alanine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/anthracene.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/arginine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/asparagine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/aspartic_acid.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/benzene.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/benzene.png" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/biphenyl.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/boc.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/boc.cml~" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/cyclobutane.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/cycloheptane.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/cyclohexane.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/cyclohexane.png" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/cyclooctane.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/cyclopentadiene-sp.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/cyclopentadiene.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/cyclopentane.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/cyclopentane.png" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/cyclopropane.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/cysteine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/cytosine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/d-fructose.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/d-glucose.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/dabcyl.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/dabsyl.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/dansyl.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/deoxyribose.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/edans.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/fmoc.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/glutamic_acid.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/glutamine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/glycine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/guanine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/histidine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/imidazole.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/imidazole.png" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/indole.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/isoleucine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/leucine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/lysine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/methionine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/naphthalene.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/nitrophenylalanine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/phenylalanine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/proline.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/purine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/pyrimidine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/ribose.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/serine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/statine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/steroid.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/threonine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/thymine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/tryptophan.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/tyrosine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/uracil.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/valine.cml" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/xdrawchem_de.qm" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/xdrawchem_en.qm" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/xdrawchem_es.qm" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/xdrawchem_fr.qm" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/xdrawchem_it.qm" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/xdrawchem_nl.qm" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/xdrawchem_pl.qm" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/xdrawchemrc" "$(DESTDIR)$(RINGDIR)"
+	-$(COPY_FILE) "ring/xdrawchemrc~" "$(DESTDIR)$(RINGDIR)"
 	
 uninstall_ringfile: 
-	-$(DEL_FILE) -r $(INSTALL_ROOT)/usr/local/lib/xdrawchem/*
-	-$(DEL_DIR) "$(INSTALL_ROOT)/usr/local/lib/xdrawchem/"
+	-$(DEL_FILE) -r $(DESTDIR)$(RINGDIR)*
+	-$(DEL_DIR) "$(DESTDIR)$(RINGDIR)"
 
-install: all install_ringfile 
+install: install_ringfile 
+	install -d $(DESTDIR)$(BINDIR)
+	install xdrawchem $(DESTDIR)$(BINDIR)
 
 uninstall: uninstall_ringfile 
 
