--- Makefile~	Mon Aug  5 18:02:38 2002
+++ Makefile	Mon Aug  5 18:10:31 2002
@@ -123,9 +123,10 @@
 moc: $(SRCMOC)
 
 install: all
-	install -m 755 xdrawchem $(INSTDIR)
-	install -d $(INSTRING)
-	install -m 644 ring/* $(INSTRING)
+	install -d $(DESTDIR)$(INSTDIR)
+	install -m 755 xdrawchem $(DESTDIR)$(INSTDIR)
+	install -d $(DESTDIR)$(INSTRING)
+	install -m 644 ring/* $(DESTDIR)$(INSTRING)
 
 tmake:
 	tmake application.pro
